gapipy/resources/base.py

```diff
@@ -78,7 +79,8 @@ def _update(self, partial=False):
 
         data = self.to_dict()
         if partial:
-            data = {k: v for k, v in data.iteritems() if self._raw_data.get(k) != v}
+            # .items isn't effecient in Python 2
+            data = {k: v for k, v in data.items() if self._raw_data.get(k) != v}
```

Comment:
That depends on what a `large dict` is. Anyway, I followed this: http://python-future.org/compatible_idioms.html. It says `inefficient` but not breaking. Our resources aren't _that_ large, nor did I read anywhere this is a breaking change. So we should be good on this one.